spring-boot-dependencies/pom.xml

```diff
@@ -2285,6 +2286,26 @@
 				<version>${jooq.version}</version>
 			</dependency>
 			<dependency>
+				<groupId>org.jetbrains.kotlin</groupId>
+				<artifactId>kotlin-reflect</artifactId>
+				<version>${kotlin.version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.jetbrains.kotlin</groupId>
+				<artifactId>kotlin-stdlib</artifactId>
+				<version>${kotlin.version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.jetbrains.kotlin</groupId>
+				<artifactId>kotlin-stdlib-jre7</artifactId>
```

Comment:
We don't support Java 7 so that should go away.

Reply:
I'm not sure I agree with that. I'd rather that the dependency management was complete.

Reply:
I am not following. When are we going to use that dependency if we don't support Java 7 anymore?

Reply:
We support Java 8 so kotlin-stdlib, kotlin-stdlib-jre7 and kotlin-stdlib-jre8 should be provided for consistent versioning. kotlin-stdlib-jre7 contains Java 7 specific methods which are needed for Java 8 support, same for kotlin-stdlib-jre8 and Java 8.